reference/SimpleCalculator/SimpleCalculator.Shared/Business/Calculator.cs

```diff
@@ -1,183 +1,238 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Linq;
 
-namespace SimpleCalculator.Business
+namespace SimpleCalculator.Business;
+
+public record Calculator
 {
-    public record Calculator
+    public static Dictionary<KeyInput, string> KeyTranslator = new()
+    {
+        { KeyInput.Multiplication, "×" },
+        { KeyInput.Division, "÷" },
+        { KeyInput.Subtraction, "−" },
+        { KeyInput.Addition, "+" },
+        { KeyInput.Back, "⌫" },
+        { KeyInput.Dot, "." },
+        { KeyInput.Clear, "C" },
+        { KeyInput.Equal, "=" },
+        { KeyInput.Percentage, "%" },
+        { KeyInput.PlusMinus, "±" },
+        { KeyInput.Zero, "0" },
+        { KeyInput.One, "1" },
+        { KeyInput.Two, "2" },
+        { KeyInput.Three, "3" },
+        { KeyInput.Four, "4" },
+        { KeyInput.Five, "5" },
+        { KeyInput.Six, "6" },
+        { KeyInput.Seven, "7" },
+        { KeyInput.Eight, "8" },
+        { KeyInput.Nine, "9" }
+    };
```

Comment:
we don't need this... you have an enum... the conditional logic should just be checking the enum value

Reply:
Changes in next commit